rector.php

```diff
@@ -40,6 +41,7 @@
         ExplicitNullableParamTypeRector::class,
     ])
     ->withSkip([
+        NewlineAfterStatementRector::class, // TODO(https://github.com/carthage-software/mago/issues/143)
```

Comment:
this is fixed. can be removed from rector.

Reply:
Good catch!